public/style.css

```
@@ -40,6 +40,7 @@
 @import url("components/chat-settings-panel.css");
 @import url("native/settings/settings-panel.css");
 @import url("native/settings/settings-config.css");
+@import url("native/scheduled-tasks/scheduled-tasks-panel.css");
```

Stylelint (17.14.0)

[error] Expected "url("native/scheduled-tasks/scheduled-tasks-panel.css")" to be ""native/scheduled-tasks/scheduled-tasks-panel.css"" (import-notation)

(import-notation)